src/app/BUILD.gn

```diff
@@ -449,6 +449,18 @@ source_set("attribute-persistence") {
   ]
 }
 
+source_set("generic-event-management") {
```

Comment:
Could we change the name to something related to the test event triggers? `generic-event-management` sounds like it should be used by majority of platforms and examples, but it's not actually about usual events management, right?